wireguard-install.sh

```diff
@@ -62,6 +81,71 @@ AllowedIPs = $CLIENT_WG_IPV4/32,$CLIENT_WG_IPV6/128" >>"/etc/wireguard/$SERVER_W
 	echo "It is also available in $HOME/$SERVER_WG_NIC-client-$CLIENT_NAME.conf"
 }
 
+function uninstall() {
+
+## To uninstall wireguard and its setup ##
+
+
+# Check OS type
+checkOS
+
+# get WireGuard interface name
+
+source /etc/wireguard/params
+
+# stop wireguard interface
+ip link set down "$SERVER_WG_NIC"	
```

Comment:
Shouldn't we disable the interface _after_ we stop wireguard service? Just asking

Reply:
I mean that wireguard uses this interface and so running `ip link set down "$SERVER_WG_NIC"` before `systemctl stop "wg-quick@$SERVER_WG_NIC"` seems wrong for me. I might be wrong though.

Reply:
Tested the scenario. Stopping the service is stopping the interface and deleting it. So There is no need to put the wgo down and delete it explicitly. Just stopping the service and disabling it from autostart at boot should be enough.

Reply:
Removed below lines and included service stop and disable line only
ip link set down "$SERVER_WG_NIC"
ip link delete "$SERVER_WG_NIC"